moment.d.ts

```diff
@@ -684,8 +684,8 @@ declare namespace moment {
   export function weekdaysMin(localeSorted: boolean, format: string): string[];
   export function weekdaysMin(localeSorted: boolean, format: string, index: number): string;
 
-  export function min(...moments: MomentInput[]): Moment;
-  export function max(...moments: MomentInput[]): Moment;
+  export function min(...moments: Moment[]): Moment;
+  export function max(...moments: Moment[]): Moment;
```

Comment:
Can you please add this overload as well?

```ts
export function min(moments: Moment[]): Moment;
```

Similarly for `max`.

(Both methods allow for passing in an array directly, but this is not reflected in the types right now.)